metadata-models/src/main/pegasus/com/linkedin/common/lineage/GenericLineage.pdl

```diff
@@ -0,0 +1,12 @@
+namespace com.linkedin.common.lineage
+
+/**
+ * Downstream lineage of a dataset
```

Comment:
Is this always Downstream Lineage? Or could we in theory use this for upstream lineage also?

Reply:
It could support upstream as well- this was copy+pasted- let me addreess